### ISSUE TITLE
Additional migration is created when `manage.py makemigrations [app name]` is run

### ISSUE THREAD

**Issue by eandersons** [stefanfoulis/django-filer#525]
Django: 1.7.8
    django-filer: 0.9.9

Hello!

When command `manage.py makemigrations [app name]` is executed, automatic migration is created for `django-filer`. This new migration alters `related_name` for the field `polymorphic_ctype` (from model `File`) to `polymorphic_filer.file_set+` (in initial migration `related_name` for this field is `polymorphic_filer.file_set`). This is causing problems when creating new environment, for example, deploying project to server.

Has anyone else faced this problem?
Any ideas how to avoid/solve this problem?

**Comment by eandersons** [comment]
Well, it seems that I found solution - I had to manually modify dependencies for the migration. I changed
```
dependencies = [
    ('filer', '0002_auto_20150508_1122'),
]
```
to
```
dependencies = [
    ('filer', '0001_initial'),
]
```

I am still interested in why this automatic migration is happening.

**Comment by shatsinn** [comment]
For the record, i observe the same as @eandersons . From #518 i conclude the issue here will be fixed by the next release. Django migrations have become much more sensitive when it comes to model changes, as discussed in e.g. https://github.com/divio/django-cms/issues/3572 (as you can see there,  @yakky is aware of the issue already)

**Comment by akx** [comment]
This is related to this django-polymorphic commit:

https://github.com/chrisglass/django_polymorphic/commit/90933184e97e1ef2b1f45e6caa4b372fa4d2420c#comments

**Comment by yakky** [comment]
Fixed by #542
